packages/core/README.md

```diff
@@ -3,11 +3,53 @@
 LoopBack makes it easy to build modern applications that require complex
 integrations.
 
+# Overview
+
 - Fast, small, powerful, extensible core
 - Generate real APIs with a single command
 - Define your data and endpoints with OpenAPI
 - No maintenance of generated code
 
-# License
+
+## Installation
+
+```shell
+$ npm install --save-dev @loopback/core
+```
+
+_This package is typically used in tests, save it to `devDependencies` via `--save-dev`._
```

Comment:
This sentence would be appropriate in `testlab` README, not here. `@loopback/core` is used in production.